src/reportSuggestionShown.ts

```diff
@@ -18,5 +22,7 @@ export default function reportSuggestionShown(
         filename: document.fileName,
       },
     });
+  } else {
+    lastShownSuggestion = null;
```

Comment:
why? if there are 3 identical suggestions you'll report the 1st and 3rd ones, because in the 2nd time you set `lastShwonSuggestion` to null, and then in the 3rd time this condition will evaluate to true: `item.new_prefix !== lastShownSuggestion?.new_prefix`

Reply:
@yonip23 is right. maybe you meant to set it to null just in case there is no item (without checking the prefixes)

Reply:
you're right, I wanted to init it in any case with the latest suggestion, modified the test to test this